recipes/mkdocstrings/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "mkdocstrings" %}
+{% set version = "0.13.6" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mkdocstrings-{{ version }}.tar.gz
+  sha256: 79e5086c79f60d1ae1d4b222f658d348ebdd6302c970cc06ee8394f2839d7c4d
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - poetry
+    - python >=3.6,<4.0
+  run:
+    - beautifulsoup4 >=4.8.2,<5.0.0
+    - mkdocs >=1.1,<2.0
+    - pymdown-extensions >=6.3,<9.0
+    - python >=3.6,<4.0
+    - pytkdocs >=0.2.0,<0.10.0
```

Comment:
I am happy with these constraints, but if you can loosen any of them, I'd recommend it.

Reply:
Except for python. Leave that one as is.

Reply:
@beckermr I adjusted those to take better match the specification. Good catch!